mvs_eland_env.yml

```diff
@@ -0,0 +1,86 @@
+name: mvs_eland_env
+channels:
+  - plotly
+  - defaults
+dependencies:
+  - _libgcc_mutex=0.1=main
```

Comment:
the only dependency which should appear here is `plotly-ocra`

Reply:
I edited the yml file to reflect your changes